packages/modeling/src/operations/extrusions/earcut/assignHoles.js

```diff
@@ -1,6 +1,6 @@
 import area from '../../../maths/utils/area.js'
-import toOutlines from '../../../geometries/geom2/toOutlines.js'
-import arePointsInside from '../../../geometries/poly2/arePointsInside.js'
+import { toOutlines } from '../../../geometries/geom2/index.js'
```

Comment:
In general, internal logic should import functions directly, which prevents circular dependencies. (It happens)

why the change?

Reply:
I was trying to be more consistent with what I have seen elsewhere in JSCAD. It seems like the preferred pattern is to use destructuring to import from index.js files. This seems better because it is using the JSCAD standard API, not making assumptions about where files are located. I can revert this change if you prefer though.

Reply:
If you look at existing imports of geometry functions inside modeling, almost all of them import from index.js. In fact `assignHoles.js` is basically the only place that imported directly. That's why I changed it.

```bash
grep -rn import.*from.*geometries packages/modeling/src
```

Reply:
Yup. My mistake.

It's the general index that internals should not use, i.e. geometries/index.js, etc.